docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-runtime.adoc

```diff
@@ -47,13 +65,23 @@ There are no previews or look ahead announcements for the upcoming `32.03` relea
 
 // |===
 
-// [#api-changes]
-// === API Changes
+[#api-changes]
+=== API Changes
 
-// [cols="30%a,70%a"]
-// |===
+[cols="30%a,70%a"]
+|===
 
-// |===
+//CWP-56590 [Doc Ticket] CWP-49617 [Eng Ticket]
+//TODO: Pending approval
+|*Report vulnerabilities using Package URL (purl) format*
+|Package details now support the pURL field to help identify an absolute path for the packages. Support was added to UI, api and twistcli.
```

Comment:
The following API responses include a new parameter, `purl`:

https://pan.dev/compute/api/get-images/ [Get Image Scan Results]

https://pan.dev/compute/api/get-registry/ [Get Registry Scan Results]

https://pan.dev/compute/api/get-scans/ [Get All CI Image Scan Results]

https://pan.dev/compute/api/get-hosts/ [Get Host Scan Results]

https://pan.dev/compute/api/get-vms/ [Get VM Image Scan Results]

https://pan.dev/compute/api/get-serverless/ [Get Serverless Function Scan Results]

The ‘purl’ field identifies the absolute path for the packages.

Reply:
done